lib/challenge.js

```diff
@@ -19,6 +19,11 @@
   // - park
 
 // Answer Starts Here
+var car = {
+  make: 'Jeep',
+  model: 'Wrangler',
+  year: 2016,
+};
```

Comment:
This is not what the question asked for, but you are assigning a mostly good object literal to `car`. That comma after `2016` is not necessary, since `year: 2016` is the final key-value pair in the object literal. The interpreter won't complain, though.